docs/Basic-Guide.md

```diff
@@ -1,105 +1,154 @@
 # Basic Guide
```

Comment:
It seems a lot of these changes are due to formatting of the markdowns. Is it possible to reduce the changes made to the docs and only include text changes ?

Reply:
Hmm, since the changes are already pushed it could be difficult. I figured that since we'd want to clean up the docs eventually I'd just do it at the same time. I agree it makes reviewing more difficult. If you feel strongly about undoing these changes, I'm happy to do it!